announcements/_posts/2019-03-01-w7-graph-databases.md

```diff
@@ -0,0 +1,21 @@
+---
+author: Jeffery Russell (@jrtechs)
+title: "Week #3: Intro to the Cloud"
+layout: post-event
+date-start: "2019-03-01 16:00"
+date-end: "2019-03-01 18:00"
+location: "GOL-2650 (Lg. DB Lab)"
+---
+
+Hey RITluggers,
+
+This week we will be talking about graph databases and how they will change the way you think about data.
+If you are interested in big data and the oodles of open source software centered around it, be sure to check out this week's meeting.
+
+We now have a #news channel in our slack where we will be aggregating all the fun linux news of the week.
```

Comment:
`s/linux/Linux`

Also, in terms of news, the `#news` channel is simply supplementary. We were/are aiming to have a News category *in* the weekly emails we send out. That way, folks can stay up to date on Linux news without coming to our meetings.

Essentially, I would like the new format of our weekly newsletters/announcements to be:
```
INTRO
---------------
Hey RITluggers! (.....)


NEWS
---------------
Current Linux news for this week


UPCOMING EVENTS
---------------
RITlug events coming up in the near future
```

Reply:
Do we have news for this week in the email?

Reply:
Also, `s/slack/Slack`

Reply:
@jrtechs I was going to get some. Got caught up in some work. If you want to send the email out tonight, then just keep the email as it is. If you want to do it tomorrow early afternoon, then I can grab some news

Reply:
@Tjzabel that would be great!